src/chat.ts

```diff
@@ -21,7 +21,7 @@ export class Chat {
 
     const prompt =
       process.env.PROMPT ||
-      'Bellow is the code patch, please help me do a brief code review, if any bug risk and improvement suggestion are welcome';
+        'Below is a code patch, please help me do a brief code review on it. Any bug risks and/or improvement suggestions are welcome:';
 
     return `${prompt}, ${answerLanguage}:
     ${patch}
```

Comment:
> "Bellow" is spelled incorrectly, it should be "Below."

Judging by the response here. It looks like it might not correctly understand that some code is being removed. Do you think this is worth open a bug report?

Reply:
Oh. Super thank you for correcting this. English is not my mother language .